README.md

```diff
@@ -37,6 +37,7 @@ Cargo requires the following tools and packages to build:
 * `cmake`
 * `pkg-config`
 * OpenSSL headers (`libssl-dev` package on ubuntu)
+* libgit2
```

Comment:
Hmm, that wasn't my experience building on OS X. Perhaps this should be an OS X specific note? Or perhaps I've missed something.